eshell-prompt-extras.el

```diff
@@ -342,9 +336,8 @@ length of PATH (sans directory slashes) down to MAX-LEN."
       (concat (epe-remote-user) "@" (epe-remote-host) " ")
       'epe-remote-face))
    (when epe-show-python-info
-     (when (fboundp 'epe-venv-p)
-       (when (and (epe-venv-p) venv-current-name)
-         (epe-colorize-with-face (concat "(" venv-current-name ") ") 'epe-venv-face))))
+     (when (and (boundp 'venv-current-name) venv-current-name)
```

Comment:
or simply `(bound-and-true-p venv-current-name)`

Reply:
@xuchunyang you're right, that's even better. As you merged already should I make a new PR or do you want to fix and ammend the last commit yourself?

Reply:
@dakra I am actually fine with the both so I will not change it myself. However, if you make a PR for this, I am happy to merge it.